python/py-bidi/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-bidi
+python.rootname     python-${python.rootname}
+version             0.4.2
+license             GNU
+maintainers         nomaintainer
+supported_archs     arm64 x86_64
+
+description         Pure python implementation of the BiDi layout algorithm
+long_description    {*}${description}
+
+homepage            https://github.com/MeirKriheli/python-bidi
+
+checksums           rmd160  e5d154ec19b81a687d1eed0ea00fb5ec23fef68d \
+                    sha256  5347f71e82b3e9976dc657f09ded2bfe39ba8d6777ca81a5b2c56c30121c496e \
+                    size    40373
+
+python.versions     38 39 310
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-wheel
+
+    depends_lib-append \
+                    port:py${python.version}-setuptools
```

Comment:
are you sure about these dependencies? Upstream lists `six` as a requirement, `wheel` shouldn't be needed I think and `setuptools` should just be a `build` dependency (I'm pretty sure with recent versions it will use `imporlib-metadata` for the entry-points and that's part of Python 3.8 and above). Please verify.

Reply:
Thanks done.